frontend/zulu/src/components/Login/profile.component.js

```diff
@@ -8,6 +8,27 @@ export default class Profile extends Component {
     this.state = { currentUser: AuthService.getCurrentUser() };
 
   }
+
+
+  onChangePassword(e) {
```

Comment:
We don't have BE support for that, though it would be a nice addition! :)

Meanwhile, this code wouldn't work (if I got your intention right - allowing the user to change his password)